### PR TITLE
Integration of OTP Verification feature with registration screen.

### PR DESCRIPTION
**Pull request by anjanibarlapati**
## 📌 What does this PR do?
- This PR integrates OTP Verification with registration feature.
- This PR modifies the registration feature to verify user details and send OTP to user's email. 
- After sending OTP successfully, user is navigated to OTP verification screen.
- Updated OTP verification feature to allow user to verify OTP in two attempts. 
- Handled all the possible edge cases in OTP verification step. When user successfully verifies OTP, will be navigated to home screen.

### 🧪 Testing
- All the code added and updated in this screen were unit test using `jest` and `RNTL` and everything is working as expected with backend.

Thank you 😊!

### CODE REVIEW

File: src/screens/OTPVerification/OtpVerification.test.tsx [hunk 1]
```diff
@@ -3,6 +3,10 @@ import React, { act } from 'react';
 import { OtpVerification } from './OtpVerification';
 import { Provider } from 'react-redux';
 import { store } from '../../redux/store';
+import * as OtpService from './OtpVerification.service';
+import EncryptedStorage from 'react-native-encrypted-storage';
+import { generateKeyPair, storeKeys } from '../../utils/keyPairs';
+import { encryptPrivateKey, decryptPrivateKey } from '../../utils/privateKey';
```
Comment (jangamkeerthi26) [inline src/screens/OTPVerification/OtpVerification.test.tsx:9]: Once check the import statements order
